addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1278,13 +1291,17 @@ export class PosStore extends Reactive {
 
             for (const order_id of order_ids_to_sync) {
                 this.db.remove_order(order_id);
+                this.syncingOrders.delete(order_id);
```

Comment:
Maybe you can handle the `syncingOrders` deletion in finally ? Like that you don't need to put it two times ?